[ERROR] 2018-10-24 07:01:09,698 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:02:09,770 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:03:09,840 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:04:09,910 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:05:09,981 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:06:10,050 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:07:10,120 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:08:10,191 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:09:10,261 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:10:10,332 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:11:10,402 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:12:10,473 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:13:10,543 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:14:10,614 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:15:10,684 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:16:10,753 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:17:10,824 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:18:10,894 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:19:10,965 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:20:11,035 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:21:11,105 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:22:11,175 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:23:11,246 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:24:11,316 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:25:11,386 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:26:11,457 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:27:11,527 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:28:11,598 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:29:11,668 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:30:11,738 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:31:11,808 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:32:11,879 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:33:11,949 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:34:12,019 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:34:43,979 | /home/vini/frappe-bench/apps/frappe/frappe/app.py:
Site: site1.local
Request Error
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/app.py", line 59, in application
    init_request(request)
  File "/home/vini/frappe-bench/apps/frappe/frappe/app.py", line 110, in init_request
    frappe.init(site=site, sites_path=_sites_path)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)
[ERROR] 2018-10-24 07:34:44,126 | /home/vini/frappe-bench/apps/frappe/frappe/app.py:
Site: site1.local
Request Error
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/app.py", line 59, in application
    init_request(request)
  File "/home/vini/frappe-bench/apps/frappe/frappe/app.py", line 110, in init_request
    frappe.init(site=site, sites_path=_sites_path)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)
[ERROR] 2018-10-24 07:35:12,089 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:36:12,157 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:37:12,227 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:38:12,297 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:39:12,367 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:40:12,438 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:41:12,508 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:42:12,579 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:43:12,650 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:44:12,721 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:45:12,791 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:46:12,862 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:47:12,932 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:48:13,002 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:49:13,073 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:50:13,143 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:51:13,213 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:52:13,282 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:53:13,352 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:54:13,423 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:55:13,493 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:56:13,563 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:57:13,633 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:58:13,704 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 07:59:13,774 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:00:13,845 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:01:13,916 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:02:13,987 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:03:14,057 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:04:14,128 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:05:14,199 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:06:14,270 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:07:14,341 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:08:14,411 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:09:14,481 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:10:14,553 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:11:14,622 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:12:14,693 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:13:14,763 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:14:14,833 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:15:14,904 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:16:14,974 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:17:15,045 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:18:15,115 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:19:15,185 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:20:15,256 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

[ERROR] 2018-10-24 08:21:15,328 | /home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Exception in Enqueue Events for Site site1.local
Traceback (most recent call last):
  File "/home/vini/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 80, in enqueue_events_for_site
    frappe.init(site=site)
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 135, in init
    local.conf = _dict(get_site_config())
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 189, in get_site_config
    config.update(get_file_json(site_config))
  File "/home/vini/frappe-bench/apps/frappe/frappe/__init__.py", line 896, in get_file_json
    return json.load(f)
  File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 16 column 1 (char 362)

